recipes/openai_ner.py

```diff
@@ -270,12 +263,29 @@ def _parse_response(self, text: str) -> List[Tuple[str, List[str]]]:
             if line and ":" in line:
                 label, phrases = line.split(":", 1)
                 label = _normalize_label(label)
-                if label in self.labels:
-                    if phrases.strip():
-                        phrases = [phrase.strip() for phrase in phrases.strip().split(",")]
-                        output.append((label, phrases))
+                if phrases.strip():
+                    phrases = [phrase.strip() for phrase in phrases.strip().split(",")]
+                    output.append((label, phrases))
```

Comment:
I removed the `if label in self.labels` check here, because we're doing the same filtering on that just below, and it saved me from passing the labels list in here too.